client/gui/ShortcutHandler.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * ShortcutHandler.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+enum class EShortcut;
+using SDL_Keycode = int32_t;
+
+class ShortcutHandler
+{
+public:
+	/// returns list of shortcuts assigned to provided SDL keycode
+	std::vector<EShortcut> translateKeycode(SDL_Keycode key);
```

Comment:
As I can imagine, return value is used to check if necessary shortcut is pressed.
If that's correct, consider `std::set<EShortcut>`

Reply:
Not exactly, it is used as list - only operation is calling function for each returned value. So no need for set (although it would also work)